package.json

```diff
@@ -60,5 +62,8 @@
   },
   "bugs": {
     "url": "https://github.com/devfolioco/react-otp-input/issues"
+  },
+  "dependencies": {
+    "docz": "^2.0.0-rc.52"
```

Comment:
We should remove docz from the dependencies and move it devDependencies.

Reply:
Everything else looks good!

Reply:
Great work. 🙌

Reply:
done